Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'cocoapods', '= 1.10.0'
```

Comment:
This is required to fix the build issue seen here: https://app.circleci.com/pipelines/github/adobe/aepsdk-edge-ios/331/workflows/6f354a9c-01b8-4341-92fb-8b7be957f743/jobs/354